### ISSUE TITLE
Animated tiles seem to just stack frames, instead of circling through them

### ISSUE THREAD

**Issue by nmfm** [bitcraft/pyscroll#50]
Animated tiles seem like they are just drawing each frame on top of the previous, without clearing the image.

I'm attaching a test Tiled map: [test.zip](https://github.com/bitcraft/pyscroll/files/7271734/test.zip), that I tested with [demo.py](https://github.com/bitcraft/pyscroll/blob/master/apps/demo/demo.py)

The flag animation has three frames: one with a blue flag, another with a red flag, and the third just the pole. Blue and red bits are always visible, and you never get the sole pole. The animation displays fine in Tiled (v1.3.3).

**Comment by bitcraft** [comment]
Thanks for including the map files and describing the problem clearly.  I looked into it, and the issue is a variation of the problem "Why are the tiles repeating while scrolling?" https://github.com/bitcraft/pyscroll#why-are-tiles-repeating-while-scrolling
A map for pyscroll *must* have a solid tile in every position of the map.  If there are positions with no tiles, then you will see problems like this.  As a test, I edited the map and added a solid tile under the animated flag and its working fine.  So for fix it, make sure you have a soldi tile under the flag... like grass, a road, dirt, etc.  In this case, I just put a while tile under it.
![image](https://user-images.githubusercontent.com/556711/135769067-fb3a4858-94bf-4e93-9711-1e7ef83cf7ed.png)
